data/data/bootstrap/baremetal/files/usr/local/bin/startironic.sh.template

```diff
@@ -3,6 +3,14 @@ set -ex
 
 . /usr/local/bin/release-image.sh
 
+grep -q "ironic.service" /etc/motd || cat <<EOF>> /etc/motd
+
+For monitoring baremetal provisioning services, you can look at ironic.service logs:
+
+  journalctl -b -f -u ironic.service
+
+EOF
+
```

Comment:
I wonder if we could achieve the same with a file e.g `data/data/bootstrap/baremetal/files/etc/motd` - not tried it but it looks like we could either copy the existing motd, or perhaps append to it?

https://github.com/openshift/installer/blob/master/pkg/asset/ignition/bootstrap/bootstrap.go#L160

https://github.com/openshift/installer/blob/master/pkg/asset/ignition/bootstrap/bootstrap.go#L303